DTALauncherStub.csproj

```diff
@@ -1,145 +1,39 @@
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{7477EAC1-2302-4311-9F7D-2D65C016C857}</ProjectGuid>
+    <TargetFramework>net7.0-windows</TargetFramework>
     <OutputType>WinExe</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <UseWindowsForms>true</UseWindowsForms>
+    <Title>DTA Launcher</Title>
+    <Description>Launcher for Dawn of the Tiberium Age CnCNet Client</Description>
+    <Company>CnCNet</Company>
+    <Product>Dawn of the Tiberium Age</Product>
+    <Copyright>Copyright © CnCNet, Rampastring 2011-2022</Copyright>
+    <Trademark></Trademark>
+    <Version>2.0.0.1</Version>
+    <FileVersion>2.0.0.1</FileVersion>
+    <InformationalVersion>2.0.0.1</InformationalVersion>
+    <AssemblyName>DTALauncherStub</AssemblyName>
     <RootNamespace>DTALauncherStub</RootNamespace>
-    <AssemblyName>DTA</AssemblyName>
-    <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
-    <TargetFrameworkProfile />
-    <PublishUrl>publish\</PublishUrl>
-    <Install>true</Install>
-    <InstallFrom>Disk</InstallFrom>
-    <UpdateEnabled>false</UpdateEnabled>
-    <UpdateMode>Foreground</UpdateMode>
-    <UpdateInterval>7</UpdateInterval>
-    <UpdateIntervalUnits>Days</UpdateIntervalUnits>
-    <UpdatePeriodically>false</UpdatePeriodically>
-    <UpdateRequired>false</UpdateRequired>
-    <MapFileExtensions>true</MapFileExtensions>
-    <ApplicationRevision>0</ApplicationRevision>
-    <ApplicationVersion>1.0.0.%2a</ApplicationVersion>
-    <IsWebBootstrapper>false</IsWebBootstrapper>
-    <UseApplicationTrust>false</UseApplicationTrust>
-    <BootstrapperEnabled>true</BootstrapperEnabled>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <PlatformTarget>AnyCPU</PlatformTarget>
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <PlatformTarget>AnyCPU</PlatformTarget>
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-  </PropertyGroup>
-  <PropertyGroup>
-    <StartupObject>DTALauncherStub.Program</StartupObject>
-  </PropertyGroup>
-  <PropertyGroup>
+    <ComVisible>false</ComVisible>
+    <CLSCompliant>false</CLSCompliant>
+    <AnalysisLevel>preview-all</AnalysisLevel>
+    <LangVersion>preview</LangVersion>
```

Comment:
I suggest we fix the version for stability.

Reply:
Replaced `preview-all` with `latest-all`. `latest` is the default for `LangVersion` so that property was removed.

Reply:
I meant that we should specify the exact version. API breaks are rare but it's still better to verify the language version change doesn't break something manually so that it doesn't automagically break.

Reply:
When omitting `LangVersion` (or using `latest`) the language version is actually fixed. `latest` means compiler version of the selected project framework. Even when installing newer SDKs the compiler version will not change.
In case you change the project framework to .NET8 you more than likely want to have access to the .NET8 compiler version as well and not be stuck with the .NET7 version.

Reply:
That's good to know, though an unintuitive naming choice from the .NET side.